packages/react-components/react-avatar/src/components/AvatarGroup/useAvatarGroup.tsx

```diff
@@ -0,0 +1,118 @@
+import * as React from 'react';
+import type { AvatarGroupProps, AvatarGroupState } from './AvatarGroup.types';
+import { getNativeElementProps, resolveShorthand } from '@fluentui/react-utilities';
+import { PopoverSurface } from '@fluentui/react-popover';
+import { Button } from '@fluentui/react-button';
+import { MoreHorizontalRegular } from '@fluentui/react-icons';
+import { avatarGroupDefaultStrings } from './AvatarGroup.strings';
+import { useId } from '@fluentui/react-utilities';
+import { getInitials } from '../../utils/getInitials';
+import { extraAvatarGroupClassNames } from './useAvatarGroupStyles';
+import { Label } from '@fluentui/react-label';
```

Comment:
_nit:_ Order alphabetically

```suggestion
import { Button } from '@fluentui/react-button';
import { MoreHorizontalRegular } from '@fluentui/react-icons';
import { Label } from '@fluentui/react-label';
import { PopoverSurface } from '@fluentui/react-popover';
import { getNativeElementProps, resolveShorthand, useId } from '@fluentui/react-utilities';
import { getInitials } from '../../utils/getInitials';
import { extraAvatarGroupClassNames } from './useAvatarGroupStyles';
import type { AvatarGroupProps, AvatarGroupState } from './AvatarGroup.types';
```